suite-native/module-accounts/src/screens/AccountDetailScreen.tsx

```diff
@@ -53,6 +54,13 @@ export const AccountDetailScreen = memo(
             [accountKey, dispatch],
         );
 
+        useEffect(() => {
+            analytics.report({
+                type: EventType.AssetDetail,
+                payload: { assetSymbol: account!.symbol, tokenSymbol },
```

Comment:
Account can possibly return undefined and this will crash the app

Reply:
fixed in 7c3f39874